modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc

```diff
@@ -68,7 +66,15 @@ spec:
 <1> `type` must be set to `vrf`.
 <2> `vrfname` is the name of the VRF that the interface is assigned to. If it does not exist in the pod, it is created.
 
-.Verify the `NetworkAttachmentDefinition` CR is successfully created
+. Apply the `SriovNetwork` changes:
```

Comment:
I think we need to change from "Apply the `SriovNetwork` changes" to "Create the `SriovNetwork` " because we create new network